### PR TITLE
aid-ter ARIA

### PR DESCRIPTION
**Pull request by FutureLearnSole**
1. As a
2. want to be able to 
3.  way as other 

_Solid is a technology, like the Web, but a new level of standard which adds to the existing protocols to make it more powerful, particularly to empower individuals at home and at work._

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<footer id="footer" class="footer">
+<footer id="footer" class="footer" role="navigation">
```
Comment (Vinnl) [inline _includes/footer.html:1]: [By default](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer), a `<footer>` has  the role [`contentinfo`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Contentinfo_role):

> The contentinfo landmark role is used to identify information repeated at the end of every page of a website, including copyright information, **navigation links**, and privacy statements. This section is commonly called a footer.

(Emphasis mine.)

Thus, I don't think we need this role here:

```suggestion
<footer id="footer" class="footer">
```

File: _includes/footer.html [hunk 2]
```diff
@@ -46,6 +45,7 @@
             src="{{site.baseurl}}/assets/img/fontawesome-free-5.11.2-web/svgs/brands/github.svg"
             alt="GitHub"
             class="brand-icon"
+	    aria-label="Github"
```
Comment (Vinnl) [inline _includes/footer.html:48]: This is already covered by [the `alt` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#Attributes).

```suggestion
```

File: _includes/footer.html [hunk 3]
```diff
@@ -55,6 +55,7 @@
             src="{{site.baseurl}}/assets/img/fontawesome-free-5.11.2-web/svgs/brands/twitter.svg"
             alt="Twitter"
             class="brand-icon"
+	    aria-label="Twitter"
```
Comment (Vinnl) [inline _includes/footer.html:58]: This is already covered by [the `alt` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#Attributes).

```suggestion
```

File: _includes/footer.html [hunk 4]
```diff
@@ -64,6 +65,7 @@
             src="{{site.baseurl}}/assets/img/fontawesome-free-5.11.2-web/svgs/brands/vimeo-v.svg"
             alt="Vimeo"
             class="brand-icon"
+	    aria-label="Vimeo"
```
Comment (Vinnl) [inline _includes/footer.html:68]: This is already covered by [the `alt` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#Attributes).

```suggestion
```

File: _includes/header.html [hunk 5]
```diff
@@ -18,6 +18,7 @@
           <a
             class="navbar-item is-size-6 is-size-5-tablet is-hidden-mobile"
             href="{{ item.link }}"
+	    aria-label="{{ item.name }}"
```
Comment (Vinnl) [inline _includes/header.html:21]: The content of this element already serves as the label, so we don't need to add a different label for screen reader users :)

```suggestion
```

File: _includes/subscribe.html [hunk 6]
```diff
@@ -1,4 +1,4 @@
-<div id="newsletter" class="section">
+<div id="newsletter" class="section" aria-label="newsletter">
```
Comment (Vinnl) [inline _includes/subscribe.html:1]: The title (`<h2>`) and contents of this `<div>` already describe it. In fact, I think (but haven't tested that) that adding a label here might even prevent the contents from being spoken by screen readers.

```suggestion
<div id="newsletter" class="section">
```

File: _includes/talk.html [hunk 7]
```diff
@@ -1,4 +1,4 @@
-<div class="talk">
+<div class="talk" aria-label="a Solid talk by {{ include.speaker }}">
```
Comment (Vinnl) [inline _includes/talk.html:1]: As above, this `<div>` is already sufficiently described by its contents.

File: _includes/breadcrumbs.html [hunk 8]
```diff
@@ -1,5 +1,5 @@
 {% if page.url != "/" %}
-  <nav id="breadcrumb" class="breadcrumb">
+  <nav id="breadcrumb" class="breadcrumb" aria-label="breadcrumbs">
```
Comment (Vinnl) [inline _includes/breadcrumbs.html:2]: I'm not sure how common the terminology "breadcrumbs" is among non-developers, but in any case, the fact that this is a navigation element should already provide enough contents for users of screen readers.

```suggestion
  <nav id="breadcrumb" class="breadcrumb">
```

**Review by Vinnl — CHANGES_REQUESTED** [review]
Thanks for caring about this! Unfortunately, when adding accessibility data, we have to be very careful about doing it correctly, or we may end up making things even worse for people relying on it. And unfortunately, most of these changes are in that category. So please take a look at my comments, so that we can merge in the remaining ones :)

**Review by FutureLearnSole — COMMENTED** [review]
_rewrite co-auth_
good ARIA bad ARIA no ARIA

File: _includes/talk.html (hunk 7 above)
Comment (FutureLearnSole) [inline _includes/talk.html:1]:
```suggestion
<div class="talk">
```

File: _includes/footer.html [hunk 9]
```diff
@@ -28,8 +28,7 @@
 
   <nav class="navbar" role="navigation" aria-label="main navigation">
     <div class="navbar-brand">
-      <a class="navbar-item" href="{{site.baseUrl}}/">
-        <img
+      <a class="navbar-item" href="{{site.baseUrl}}/"> <img
```
Comment (FutureLearnSole) [inline _includes/footer.html:31]:
```suggestion
      <a class="navbar-item" href="{{site.baseUrl}}/">
        <img
```

File: _includes/subscribe.html [hunk 10]
```diff
@@ -34,4 +34,4 @@ <h2 class="title">
       </div>
     </div>
   </form>
-</div>
\ No newline at end of file
+</div>
```
Comment (FutureLearnSole) [inline _includes/subscribe.html:37]:
```suggestion
</div>\ No newline at end of file
```

File: _includes/talk.html [hunk 11]
```diff
@@ -52,4 +52,4 @@
       {% endif %}
     </aside>
   </div>
-</div>
\ No newline at end of file
+</div>
```
Comment (FutureLearnSole) [inline _includes/talk.html:55]:
```suggestion
</div>\ No newline at end of file
```

File: _includes/toc-apps.html [hunk 12]
```diff
@@ -86,4 +86,4 @@
       </li>
     {% endif %}
   {% endfor %}
-</ul>
\ No newline at end of file
+</ul>
```
Comment (FutureLearnSole) [inline _includes/toc-apps.html:89]:
```suggestion
</ul>\ No newline at end of file
```

File: _includes/toc-pod-server.html [hunk 13]
```diff
@@ -15,4 +15,4 @@
       </ul>
     </li>
   {% endfor %}
-</ul>
\ No newline at end of file
+</ul>
```
Comment (FutureLearnSole) [inline _includes/toc-pod-server.html:18]:
```suggestion
</ul>\ No newline at end of file
```

File: _includes/subscribe.html [hunk 14]
```diff
@@ -34,4 +34,4 @@ <h2 class="title">
       </div>
     </div>
   </form>
-</div>
\ No newline at end of file
+</div>\ No newline at end of file
```
Comment (FutureLearnSole) [inline _includes/subscribe.html:37]:
```suggestion
</div><EOL>
```

File: _includes/subscribe.html [hunk 15]
```diff
@@ -34,4 +34,4 @@ <h2 class="title">
       </div>
     </div>
   </form>
-</div>
\ No newline at end of file
+</div><EOL>
```
Comment (FutureLearnSole) [inline _includes/subscribe.html:37]:
```suggestion
</div><CR><LF>
```

File: _includes/talk.html [hunk 16]
```diff
@@ -52,4 +52,4 @@
       {% endif %}
     </aside>
   </div>
-</div>
\ No newline at end of file
+</div>\ No newline at end of file
```
Comment (FutureLearnSole) [inline _includes/talk.html:55]:
```suggestion
</div>
```

File: _includes/toc-apps.html [hunk 17]
```diff
@@ -86,4 +86,4 @@
       </li>
     {% endif %}
   {% endfor %}
-</ul>
\ No newline at end of file
+</ul>\ No newline at end of file
```
Comment (FutureLearnSole) [inline _includes/toc-apps.html:89]:
```suggestion
</ul>;\n
```

File: _includes/toc-pod-server.html [hunk 18]
```diff
@@ -15,4 +15,4 @@
       </ul>
     </li>
   {% endfor %}
-</ul>
\ No newline at end of file
+</ul>\ No newline at end of file
```
Comment (FutureLearnSole) [inline _includes/toc-pod-server.html:18]:
```suggestion
</ul>;\n
```

File: _includes/subscribe.html [hunk 19]
```diff
@@ -34,4 +34,4 @@ <h2 class="title">
       </div>
     </div>
   </form>
-</div>
\ No newline at end of file
+</div><CR><LF>
```
Comment (FutureLearnSole) [inline _includes/subscribe.html:37]:
```suggestion
</div>;\n
```

**Comment by Vinnl** [comment]
Thanks @FutureLearnSole, I've fixed the newline issues and resubmitted your PR in #349, which should be reviewed and merged soon :) Thanks!

**Pull request closed without merging**